sceptre/config/reader.py

```diff
@@ -422,8 +419,8 @@ def _render(self, directory_path, basename, stack_group_config):
         :type directory_path: str
         :param basename: The filename of the config file
         :type basename: str
-        :param stack_group_config: The loaded config file for the StackGroup
-        :type stack_group_config: dict
+        :param config_group: The loaded config file for the StackGroup
+        :type config_group: dict
```

Comment:
I've done a bit of renaming of variables to ensure that `stack_group_config` is used only once.